Bump docs-theme-extras to v0.1.21, add pipefail to framework-tests, added descriptions (#36)
agentregistry-dev/website: content/docs/agents/publish.md

---
title: Publish
weight: 20

> description: "Build and publish your agent image to agentregistry with the arctl agent build and publish commands."

Reviewer: ```suggestion
description: "Add your agent to the registry catalog."
```
